SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp

```diff
@@ -228,6 +228,7 @@ TArray<Worker_ComponentId> InterestFactory::CreateServerNonAuthInterestResultTyp
 
 	// Add all data and handover components
 	ResultType.Append(ClassInfoManager->GetComponentIdsForComponentType(ESchemaComponentType::SCHEMA_Data));
+	ResultType.Append(ClassInfoManager->GetComponentIdsForComponentType(ESchemaComponentType::SCHEMA_OwnerOnly));
```

Comment:
cc @samiwh and @m-samiec we discussed this and I think the outcome was that servers should see all three types currently, please correct me if I am wrong

Reply:
This is a grey area. The reason I believe the non-auth server *should* see the owner-only data is so if it becomes authoritative over it, it has that data ready, instead of receiving it after the fact.

Reply:
A non-auth server may still want to interact with a pawn based on that data though. You might need to use data, that other clients shouldn't be aware of, to identify the correct pawn to send a cross-server RPC to. If we want to enable such things then I'd be in favour of making owner only available on non-auth servers.
The only real advantage I see to making owner only stuff not visible to them is if we expect that the state is large and actually costs something significant to send. Similar to the argument about the different types of server-only state we could have (handover-only vs not-client data). However for owner stuff I don't imagine you'd have as large state so I'd rather add another data type for that as an optimisation if needed.

Generally we currently have 3 different types of data but 6 different configurations of access we could do.
I think the way we cover the most useful ground without having to make new replication types is to have:
1. Visible on all (data)
2. Visible on auth server + non auth server (handover)
3. Visible on auth server + non auth server + owning client (owner)

Then we could add the following if needs be:
4. auth server only (what I tend to think of as "handover" state)
5. auth server + owner only (optimised form of owner if there is large state only the client needs)
6. auth server + clients (maybe useful as visual only data)

Reply:
However you could also duplicate state into owner and server-only state and then make owner state not visible to non-auth servers. But this sounds more complicated.

Reply:
sounds like for now it needs to see all 3, so leaving it